soc/riscv32/litex-vexriscv/dts_fixup.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* UART 0 */
+#define CONFIG_UART_0_BASE_ADDR		DT_LITEX_UART0_E0001800_BASE_ADDRESS
```

Comment:
We moved away from using CONFIG_ prefixed defines.  I'd suggest maybe seeing if you can just use the standard generated defines instead of needing a dts_fixup.h.

Reply:
We'll use the generated defines directly.

Reply:
It's done.